### DISCUSSION TITLE
Be careful about going to Go 1.21, for breaking backwards compatibility

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#3250] [Q&A] [answered]
@ganigeorgiev I was recently caught off guard by a very unusual backwards incompatibility in XML handling.  It broke the Excel package.  So just to let you know that if you don't need 1.21 then perhaps you could wait a bit.

**Comment by ganigeorgiev** [comment, marked as answer]
Seems to have been addressed in https://go-review.googlesource.com/c/go/+/466295.
It will be fixed with the next minor release, at least based on the [related GitHub issue](https://github.com/golang/go/issues/61881#issuecomment-1693909801).
